### PR TITLE
refact: refact rpc service listen method to read to easy.

### PR DESCRIPTION
**Pull request by resoliwan**
Preserve same functionality.
Break long function to small logical functions.

### CODE REVIEW

File: src/services/rpc-service.ts [hunk 1]
```diff
@@ -58,6 +58,14 @@ export enum RpcHookType {
   POST_RPC_ERROR
 }
 
+export interface RpcEntrites {
```
Comment (heycalmdown) [inline src/services/rpc-service.ts]: typo here - Entrites -> Entities

File: src/services/rpc-service.ts [hunk 2]
```diff
@@ -190,82 +192,11 @@ export default class RPCService {
...
-      await Promise.all(_.map(this.rpcEntities, async ({ type, handler, rpcOptions }, rpcName: string) => {
-        await Bluebird.each(_.range(16), async shard => {
-          const RPC_QUEUE_NAME = this.makeRequestQueueName(shard);
-          await channel.bindQueue(RPC_QUEUE_NAME, rpcName, '' + shard);
-        });
-      }));
-    });
+    const queues = _.map(_.range(RPC_QUEUE_DISTRIB_SIZE), no => (`rpc.req.${this.serviceName}.${no}`));
+    await this.assertQueues(queues);
+    await this.assertExchages(this.rpcEntities);
```
Comment (heycalmdown) [inline src/services/rpc-service.ts]: Exchages -> Exchanges

File: src/services/rpc-service.ts [hunk 3]
```diff
@@ -190,82 +192,11 @@ export default class RPCService {
...
+    await this.bindQueuesToExchages(queues, this.rpcEntities);
```
Comment (heycalmdown) [inline src/services/rpc-service.ts]: Exchages -> Exchanges

File: src/services/rpc-service.ts [hunk 4]
```diff
@@ -540,4 +467,95 @@ export default class RPCService {
...
+      await Promise.all(_.map(bindInfos, async ({queue, rpcName, routingKey}) => {
+        await channel.bindQueue(queue, rpcName, routingKey);
+      }));
+    });
+  }
+
+  private async startConsumingQueues(queues: string[]): Promise<void> {
+    await this.consumerChannelPool.usingChannel(async channel => {
+      await Bluebird.each(queues, async queue => {
+        const consumerInfo =  await this.startConsumingQueue(queue);
```
Comment (heycalmdown) [inline src/services/rpc-service.ts]: double whitespace

Comment (resoliwan) [reply src/services/rpc-service.ts]: Fix all typos.

Thank you for careful review.

**Pull request merged**